projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Additional wpcom_admin_interface option on settings.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Add the Admin Interface Style setting on the General settings page.
+ * This setting allows users to switch between the classic WP-Admin interface and the WordPress.com legacy dashboard.
+ * The setting is stored in the wpcom_admin_interface option.
+ * The setting is displayed only if the has the wp-admin interface selected.
+ */
+function wpcomsh_wpcom_admin_interface_settings_field() {
+	add_settings_field( 'wpcom_admin_interface', null, 'wpcom_admin_interface_display', 'general', 'default' );
```

Comment:
You can make Phan (pdWQjU-Jb-p2) happy like this, but we'll want to test it again to be sure it's still working.

```suggestion
	add_settings_field( 'wpcom_admin_interface', '', 'wpcom_admin_interface_display', 'general', 'default' );
```

Reply:
Thanks for this and :point_down: